app/src/main/java/com/github/alfabravo2013/readyforexams/presentation/create/CreateFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.alfabravo2013.readyforexams.presentation.create
+
+import android.os.Bundle
+import android.view.View
+import com.github.alfabravo2013.readyforexams.R
+import com.github.alfabravo2013.readyforexams.databinding.FragmentCreateBinding
+import com.github.alfabravo2013.readyforexams.presentation.BaseFragment
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class CreateFragment : BaseFragment(R.layout.fragment_create) {
+    private val viewModel: CreateViewModel by viewModel()
+    private lateinit var binding: FragmentCreateBinding
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        binding = FragmentCreateBinding.bind(view)
+        setToolbarTitle(requireContext().getString(R.string.create_title_text))
+
+        val adapter = TaskAdapter()
+
+        with(binding) {
+            createItemsRecyclerView.adapter = adapter
+
+            createItemAddButton.setOnClickListener {
+                var item = createTaskItemEditText.text.toString()
+                if (item.isBlank()) item = "New task"
```

Comment:
Remember: try to avoid logic inside the UI. If you want to check for non-null edittext text item, pass the text to the ViewModel, check there if it s not null and then come back do the UI and add the item in the adapter.